### ISSUE TITLE
Anchor footer content at bottom of screen

### ISSUE THREAD

**Issue by zachkirsch** [fern-api/fern#4422]
On short docs pages, I think the "footer content" (like `was this page helpful` and `built with fern`) should be at the bottom of the screen.

<img width="1512" alt="Screenshot 2024-08-26 at 11 11 29 AM" src="https://github.com/user-attachments/assets/98d7aa00-07d6-4d70-97b4-d52700e5ce1c">

**Comment by dannysheridan** [comment]
That's a great idea. Assigning to @chdeskur to see this through!

**Comment by dannysheridan** [comment]
Agreed with the feedback, but low enough priority that we're not going to get to this for now.

**Comment by chdeskur** [comment]
This is now the default layout styling
